sources/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ set(od_src
     hgram.c
```

Comment:
Don't we want to remove hgram code? It's obsolete now?

Reply:
Deleted